tools/reproduce_comparison/Jenkinsfile

```diff
@@ -37,7 +38,12 @@ pipeline {
                         if (params.platform != 'linux') {
                             "prep${platform}" ('url1Dir1', 'url1Dir2')
                         }
-                        def retVal = sh returnStatus: true, script: 'diff -q -r url1Dir1 url1Dir2'
+                        excludeString=''
```

Comment:
hum. I think here is groovy file not a shell script.

some snippet like this (not verified with syntax)
```
def excludeStrings = "file1;file2"
def excludeFlags =""
def excludeStringList = excludeStrings.split(';')
for(String excludeString : excludeStringList )
  excludeFlags="${excludeFlags} --exclude=${excludeString}"
​println "DEBUG: ${excludeFlags}​​​​​​​​​​​​​​​​"
def retVal = sh returnStatus: true, script: 'diff -q -r url1Dir1 url1Dir2 ${excludeFlags}'
```

Reply:
Ugh, I was honestly getting a lil confused since its been months I have not touched jenkins, let me fix it.

Reply:
Done @zdtsw please check again